tests/mpi/parallel_block_vector_04.cc

```diff
@@ -0,0 +1,199 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+// check global reduction operations (norms, operator==, operator!=) with the
+// parallel block vector class. This is the same as parallel_vector_01 except
+// that we use the new 'add' function to set the values in the block vector
+// (parallel_vector_01 uses vector assignment).
```

Comment:
please be more explicit when describing the way elements are added: I would prefer 'we add several elements coming from an std::vector to the block vector'. In particular, I would avoid 'new' in this context... ;-)